src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,26 @@
+package calculator
+
+private typealias OperatorPair = Pair<IntArithmetics, Int>
+
+class Calculator(private val input: String) {
+    fun calculate(): Int {
+        val list = input.split(" ")
+        val identity = list[0].toInt()
+        return list.subList(1, list.size)
+            .chunked(2)
+            .map { chunk2Pair(it) }
+            .fold(identity) { acc, curr -> accumulate(acc, curr) }
```

Comment:
```suggestion
            .map(this::chunk2Par)
            .fold(identity)(this::accumulate)
```

자바의 람다처럼 위와 같은 문법으로 표현을 해보려고 했는데 쉽지 않군요.
혹시 코틀린에서 다른 객체를 람다로 참조하려 하면 어떻게 깔끔하게 참조할 수 있는지 아시나요?

아래와 같은 시도도 해보았는데, 안된다고 막히더군요.

```kotlin
            .map {
                val arithmetic = IntArithmetics.from(it[0])
                val number = it[1].toInt()
                return Pair(arithmetic, number)
            }
```